package.json

```diff
@@ -10,6 +10,7 @@
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
     "lint": "prettier --check . && eslint .",
+    "lint:fix": "prettier --write . && eslint --fix .",
```

Comment:
is there a reason to have both lint:fix and format?

Reply:
I've often added both just to make it easy to find. I tend toward `format` (Rust and Go style perhaps?), but others have tended toward `lint:fix`. I'm find with standardizing on one or the other.

Reply:
eh.
`format` was already there, but `lint:fix` was missing. Maybe `lint:fix` should *only* fix lint-y problems?